lib/lightstep/reporter.rb

```diff
@@ -29,8 +30,7 @@ def initialize(max_span_records:, transport:, guid:, component_name:)
       reset_on_fork
 
       at_exit do
-        @quit_signal << true
-        @thread.join
+        flush
```

Comment:
Minor nit: would actually prefer we don't do this "for now" – the thinking being that it blocks the user code, and that they can explicitly flush if they want that behavior.